CMakeLists.txt

```diff
@@ -8,6 +8,10 @@ project(crow-translate
 )
 
 set(APPLICATION_NAME "Crow Translate")
+set(COMPANY_NAME "Crow Translate")
```

Comment:
Please just duplicate `APPLICATION_NAME` since there is not company.

Reply:
OK: Pelase change it.